src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy

```diff
@@ -590,8 +590,8 @@ class LeVADocumentUseCase extends DocGenUseCase {
                     probabilityOfOccurrence: probabilityOfOccurrence ? probabilityOfOccurrence."short" : "None",
                     severityOfImpact: severityOfImpact ? severityOfImpact."short" : "None",
                     probabilityOfDetection: probabilityOfDetection ? probabilityOfDetection."short" : "None",
-                    riskPriority: riskPriority ? riskPriority.value : "N/A",
-                    riskPriorityNumber: r.riskPriorityNumber ?: "N/A",
+                    riskPriority: (riskPriority != null) ? riskPriority.value : "N/A",
```

Comment:
If you want to get a text value for the riskPriority, you need to use either .text (long) or .short (short). It is similar to the probabilityOfDetection, for example. Please, note that one of the enum values for the riskPriority, concretely riskPriority.value == 0, has short "N" and text "N/A". This may create confusion with the "N/A" that would be shown if riskPriority is null.
Not very important, but the (riskPriority != null) ? ... can perfectly be left as before riskPriority ? ...
For consistence with the other enums that are shown here, I suggest:
riskPriority = riskPriority."short" : "N/A"